### PR TITLE
custom selectors for overview page

### PR DESCRIPTION
**Pull request by taapsa-visma**

### CODE REVIEW

File: configs/maventa.json [hunk 1]
```diff
@@ -1,22 +1,32 @@
 {
   "index_name": "maventa",
   "start_urls": [
-    "https://documentation.maventa.com/"
+    "https://documentation.maventa.com/",
```
Comment (s-pace) [inline configs/maventa.json:4]: Should be the last as it will always match first all URLs

File: configs/maventa.json [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
+      "lvl1": ".col-sm-9 h2",
+      "lvl2": ".col-sm-9 h3",
+      "lvl3": ".col-sm-9 h4",
+      "lvl4": ".col-sm-9 h5, .col-sm-9 summary",
+      "lvl5": ".col-sm-9 h6",
+      "text": ".col-sm-9 p, .col-sm-9 li"
+    },
+    "overview": {
+      "lvl0": ".col-sm-9 h1",
+      "lvl1": ".col-sm-9 h2"
```
Comment (s-pace) [inline configs/maventa.json:25]: You need to define all selectors within a set. Please add other lvls

File: configs/maventa.json (hunk 1 above)
Comment (taapsa-visma) [reply configs/maventa.json]: moved to last

File: configs/maventa.json (hunk 2 above)
Comment (taapsa-visma) [reply configs/maventa.json]: page contains only h1 and h2 and text. text added

**Pull request merged**